packages/input/src/LionInput.js

```diff
@@ -20,6 +20,7 @@ export class LionInput extends LionField {
       readOnly: {
         type: Boolean,
         attribute: 'readonly',
+        reflect: true,
```

Comment:
should be a `fix(field)` right?

Reply:
Hmm yeah I hesistated because it's not linked to any existing issue, but I suppose we can be quite sure this is a fix and needs a release. Amended it just now